Cargo.toml

```diff
@@ -75,11 +77,11 @@ hyper-util = { version = "0.1.3", features = ["server"] }
 http-body-util = "0.1.1"
 
 [features]
-# Ideally network-interface would be disabled by default on certain platforms
-# However: https://github.com/rust-lang/cargo/issues/1197
```

Comment:
I think this comment still applies. Maybe we can also include a link to the xh issue with the NetBSD problems?